src/composables/useColourScale.ts

```diff
@@ -0,0 +1,81 @@
+import { Ref, computed } from "vue";
+import { storeToRefs } from "pinia";
+import * as d3ScaleChromatic from "d3-scale-chromatic";
+import { FeatureIndicatorValues, IndicatorValue } from "../resourceTypes";
+import { useAppStore } from "../stores/appStore";
+
+interface IndicatorRange {
+    min: number;
+    max: number;
+    range: number;
+}
+
+export const useColourScale = (selectedIndicators: Ref<Dict<FeatureIndicatorValues>>) => {
+    // TODO: we currently just scale colours to min and max in data, but
```

Comment:
I suspect this will be the desired behaviour, possibly right off the bat. Prevelance will naturally scale from 0 to 1 (though they may want more detail at the lower end). FOI is harder because  it's effectively a rate and so has no upper bound. We'd need to make sure that we're scaling against the largest value globally (so across all ADM2 units in all data sets) and that will be easiest to get as part of some sort of processing.

Can be changed later though!